pyqmc/func3d.py

```diff
@@ -5,14 +5,14 @@
 to any variational parameters the funtion has.
```

Comment:
Why are we changing the func3d interface for just one function here? It may be better, maybe not but I'm confused about this.

Does this need to be done as part of the Jastrow upgrade?

Reply:
Almost all the changes are in the docstrings. There are really very few changes in the code. Only the gaussian basis functions needed to be changed; the others already have this behavior. My reason for changing it is that it removes a bunch of reshapes in jastrowspin.py, making the code shorter and more readable. I don't know if it actually makes it faster. I meant to make this change after this pull request but already made it before fixing a bug. I can separate it out again if it's better for organization.